src/widgets/modalChildren/PatientEditModal.js

```diff
@@ -23,51 +23,59 @@ export const PatientEditModalComponent = ({
   surveyForm,
   onUpdateForm,
   nameNoteIsValid,
-}) => (
-  <FlexRow style={{ flexDirection: 'column' }} flex={1}>
-    <FlexRow flex={1}>
-      <FormControl
-        canSave={surveySchema ? nameNoteIsValid : true}
-        isDisabled={isDisabled}
-        onSave={onSaveForm}
-        onCancel={onCancel}
-        inputConfig={inputConfig}
-        showCancelButton={false}
-        showSaveButton={false}
-      />
-      {surveySchema && surveyForm && (
-        <View style={styles.formContainer}>
-          <JSONForm
-            surveySchema={surveySchema}
-            formData={surveyForm}
-            onChange={({ formData }, validator) => {
-              onUpdateForm(formData, validator);
-            }}
-          >
-            <></>
-          </JSONForm>
-        </View>
-      )}
-    </FlexRow>
-    <FlexRow flex={0} style={{ justifyContent: 'center' }}>
-      <View style={styles.buttonsRow}>
-        <PageButton
-          onPress={onSaveForm}
-          style={styles.saveButton}
-          isDisabled={surveySchema && surveyForm ? !nameNoteIsValid : isDisabled}
-          textStyle={styles.saveButtonTextStyle}
-          text={generalStrings.save}
-        />
-        <PageButton
-          onPress={onCancel}
-          style={styles.cancelButton}
-          textStyle={styles.cancelButtonTextStyle}
-          text={modalStrings.cancel}
+  canSaveForm,
```

Comment:
off road: The patient edit modal wasn't updating the save button when the form was invalid

Reply:
oh yes. I did wonder about that. thanks!